### PR TITLE
New header component and some foundation work around stylesheets

### PR DESCRIPTION
**Pull request by banciur**
Quick push so scss things are there.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "react-hot-loader": "next",
     "sass-loader": "^6.0.5",
     "style-loader": "^0.18.1",
+    "url-loader": "^0.5.8",
```
Comment (recmo) [inline package.json:44]: Where do we need this?

Comment (banciur) [reply package.json:44]: Is it used in [ webpack configuration](https://github.com/Neufund/generic-ico/blob/basic_layouts/webpack.config.js#L62)

This affects style sheets - [here](https://github.com/Neufund/generic-ico/blob/basic_layouts/src/components/HeaderInitial.scss#L17) 
In reality files if they are small enough (its configurable) they will be embedded into final js file.

File: public/index.html [hunk 2]
```diff
@@ -14,9 +14,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>To do</title>
-    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Material+Icons|Roboto|Montserrat:300,400,500,700" rel="stylesheet">
```
Comment (recmo) [inline public/index.html]: We should not include external resources without integrity marks (security reasons). For google fonts that won't work. I think we should bundle fonts, but let's make that a separate PR.

[No action required]

File: src/views/LayoutEntry.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import HeaderInitial from '../components/HeaderInitial';
+import Footer from '../components/Footer';
+
+const Layout = ({ children }) => (<div>
```
Comment (recmo) [inline src/views/LayoutEntry.js]: Rename to `LayoutEntry` to match file name

**Review by recmo — CHANGES_REQUESTED** [review]

**Review by recmo — APPROVED** [review]
